gradle/libs.versions.toml

```diff
@@ -1,7 +1,7 @@
 [versions]
 accompanist = "0.32.0"
 androidGradlePlugin = "8.1.2"
-androidx-activity-compose = "1.8.0-rc01"
+androidx-activity-compose = "1.9.0-alpha01"
```

Comment:
What API do you need in `androidx-activity-compose`? Do most snippets require this dependency or just one or two?

If most snippets require it, then I think `latest` is the correct branch to add these snippets to. Otherwise, if most snippets don't need it, then `main` would be a better target branch to merge into.